crates/next-core/src/next_server_component/server_component_module.rs

```diff
@@ -73,7 +73,7 @@ impl Module for NextServerComponentModule {
     async fn additional_layers_modules(self: Vc<Self>) -> Result<Vc<Modules>> {
         let base_ident = self.ident();
         let ssr_entry_module = ResolvedVc::upcast(
-            IncludeIdentModule::new(base_ident.with_modifier(client_modules_ssr_modifier()))
+            IncludeIdentModule::new(base_ident.with_modifier(ssr_modules_modifier()))
```

Comment:
should it still use `IncludeIdentModule`?

Reply:
That will be removed anyway once we port over the global module ids. That's the only place where `additional_layers_modules` is used.